src/assets/content/dictionary/technical-debt-1.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Technical Debt
-definition:
-sources: []
+definition: Technical debt, also known as code debt, refers to the cost or consequences of prioritizing speed or ease of implementation over quality. In software development, this can mean making coding or design decisions that will need to be addressed in the future in order to get a viable solution into production sooner. While often necessary in the short term, this debt can add up and become more problematic or expensive over time.
+- sourceurl: https://enterprisersproject.com/article/2020/6/technical-debt-explained-plain-english
```

Comment:
Take a look at the example again: https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md#tech-dictionary-example

You see that you need to keep the line 
```- sources:```